include/zephyr/drivers/video.h

```diff
@@ -1377,6 +1338,15 @@ int video_transfer_buffer(const struct device *src, const struct device *sink,
  */
 #define VIDEO_PIX_FMT_GREY VIDEO_FOURCC('G', 'R', 'E', 'Y')
 
+
+/**
+ * @code{.unparsed}
+ *   0                   1                   2
+ * | 0000yyyy 0000Yyyy | 0000yyyy 0000Yyyy | 0000yyyy 0000Yyyy | ...
```

Comment:
Some small improvement would be to use 4 pixels: all formats show how to encode 4 pixels on purpose to give an estimation of how large the data is in RAM at first glance.

```suggestion
 *   0                   1                   2                   3
 * | 0000yyyy 0000Yyyy | 0000yyyy 0000Yyyy | 0000yyyy 0000Yyyy | 0000yyyy 0000Yyyy | ...
```

Thanks for adding the doc!

Reply:
done